.github/workflows/accessibility.yml

```diff
@@ -0,0 +1,52 @@
+name: pa11y tests
+
+on: [pull_request]
+
+jobs:
+  build:
+    name: Building site and running pa11y-ci tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout source.
+        uses: actions/checkout@v5
+
+      - name: Install jekyll site dependencies.
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1.4
```

Comment:
instead add a `.ruby_version` and have it use that, so that other devs will match what you expect them to use